src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -356,6 +356,7 @@ class HttpGoogleServicesDAO(implicit val system: ActorSystem, implicit val mater
                   val extReq = Get(gcsApiUrl)
 
                   userAuthedRequest(extReq)(userAuthToken).map { proxyResponse =>
+                    proxyResponse.discardEntityBytes()
                     proxyResponse.header[`Content-Type`] match {
                       case Some(ct) =>
                         RequestCompleteWithHeaders(proxyResponse, `Content-Type`(ct.contentType))
```

Comment:
this is surprising to me ... sending the response object to the client does not consume the entity?

Reply:
I believe you are right, otherwise we would be seeing this in other places that I have not seen it